### ISSUE TITLE
Running inline results in TypeError

### ISSUE THREAD

**Issue by thirdwheel** [Yahweasel/nodejs-server-pages#3]
Seems it must be in a file, but surely running it inline like this is a suitable use case if this is all you're doing:

```
aaron_mason@hesvws3:~/njsp-test$ node -e 'require("nodejs-server-pages").createServer({port: 8090, ip: "0.0.0.0"});'
/home/aaron_mason/njsp-test/node_modules/nodejs-server-pages/njsp.js:54
    nodePath += require.main.paths.join(":");
                             ^

TypeError: Cannot read property 'paths' of undefined
    at /home/aaron_mason/njsp-test/node_modules/nodejs-server-pages/njsp.js:54:30
    at Object.<anonymous> (/home/aaron_mason/njsp-test/node_modules/nodejs-server-pages/njsp.js:56:3)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:93:18)
    at [eval]:1:1
    at Script.runInThisContext (vm.js:134:12)
```

**Comment by Yahweasel** [comment]
I didn't anticipate the case of not having a main module. I use require.main to get the paths so that spawned worker threads have access to all the modules that the njsp script itself has access to, instead of only the prerequisites of njsp itself. I'll update that to (require.main || module) some time tonight, and that should allow either mode to work.

**Comment by Yahweasel** [comment]
Fixed and published as v1.2.1.
